docs/api.rst

```diff
@@ -300,3 +296,10 @@ changed meanwhile, a ``412 Precondition failed`` error is returned.
 
 :note:
     As mentionned in the *Validation section*, an article status cannot take the value ``2``.
+
+
+Conflicts
+---------
+
+If changing the article ``resolved_url`` violates the unicity constraint, a
+``409 Conflict`` error response is returned (see :ref:`error channel <_error-responses>`).
```

Comment:
So the unicity is only on ``resolved_url`` not on ``url``? How does that works with a worked populating the ``resolved_url``?